scripts/install_third_party_libs.py

```diff
@@ -0,0 +1,272 @@
+            os.chdir(common.CURR_DIR)
+
+        # Move the build directory to the static resources folder.
+        shutil.copytree(
+            os.path.join(
+                common.OPPIA_TOOLS_DIR, 'skulpt-0.10.0/skulpt/dist/'),
+            os.path.join(common.THIRD_PARTY_DIR, 'static/skulpt-0.10.0'))
+
+
+def maybe_install_dependencies(
+        skip_installing_third_party_libs, run_minified_tests):
+    """Parse additional command line arguments."""
+    if skip_installing_third_party_libs is False:
+        # Install third party dependencies.
+        main(args=[])
+        # Ensure that generated JS and CSS files are in place before running the
+        # tests.
+        python_utils.PRINT('Running build task with concatenation only')
+        build.main(args=[])
+
+    if run_minified_tests is True:
+        python_utils.PRINT(
+            'Running build task with concatenation and minification')
+        build.main(args=['--prod_env'])
+
+
+def ensure_pip_library_is_installed(package, version, path):
+    """Installs the pip library after ensuring its not already installed.
+
+    Args:
+        package: str. The package name.
+        version: str. The package version.
+        path: str. The installation path for the package.
+    """
+    python_utils.PRINT(
+        'Checking if %s is installed in %s' % (package, path))
+
+    exact_lib_path = os.path.join(path, '%s-%s' % (package, version))
+    if not os.path.exists(exact_lib_path):
+        python_utils.PRINT('Installing %s' % package)
+        pip_install(package, version, exact_lib_path)
+
+
+def main(args=None):
+    """Install third-party libraries for Oppia."""
+    setup.main(args=[])
+    setup_gae.main(args=[])
+    pip_dependencies = [
+        ('pylint', '1.9.4', common.OPPIA_TOOLS_DIR),
+        ('Pillow', '6.0.0', common.OPPIA_TOOLS_DIR),
+        ('pylint-quotes', '0.1.8', common.OPPIA_TOOLS_DIR),
+        ('webtest', '2.0.33', common.OPPIA_TOOLS_DIR),
+        ('isort', '4.3.20', common.OPPIA_TOOLS_DIR),
+        ('pycodestyle', '2.5.0', common.OPPIA_TOOLS_DIR),
+        ('esprima', '4.0.1', common.OPPIA_TOOLS_DIR),
+        ('browsermob-proxy', '0.8.0', common.OPPIA_TOOLS_DIR),
+        ('selenium', '3.13.0', common.OPPIA_TOOLS_DIR),
+        ('PyGithub', '1.43.7', common.OPPIA_TOOLS_DIR),
+    ]
+
+    for package, version, path in pip_dependencies:
+        ensure_pip_library_is_installed(package, version, path)
+
+    # Download and install required JS and zip files.
+    python_utils.PRINT('Installing third-party JS libraries and zip files.')
+    install_third_party.main(args=[])
+
+    # Install third-party node modules needed for the build process.
+    subprocess.call([common.NPM_PATH, 'install', '--only=dev'])
+    # This line removes the 'npm ERR! missing:' messages. For reference, see
+    # this thread: https://github.com/npm/npm/issues/19393#issuecomment-
+    # 374076889.
+    subprocess.call([common.NPM_PATH, 'dedupe'])
+
+    install_skulpt(args)
```

Comment:
Let's parse the args in `main` and pass the parsed data into `install_skulpt`.

Reply:
Done